### PR TITLE
no `import *`.  not even once

### PR DESCRIPTION
**Pull request by gforsyth**
"I've seen things you people wouldn't believe. Attack ships on fire off the shoulder of Orion. I watched C-beams glitter in the dark near the Tannhäuser Gate. All those moments will be lost in time, like tears...in...rain. Time to die." - `import *`

### CODE REVIEW

**Comment by scopatz** [comment]
"It's too bad she won't live! But then again, who does?"

File: xonsh/tokenize.py [hunk 1]
```diff
@@ -24,7 +24,15 @@
 from itertools import chain
 import re
 import sys
-from token import *
+from token import (AMPER, AMPEREQUAL, ASYNC, AT, AWAIT, CIRCUMFLEX,
```
Comment (scopatz) [inline xonsh/tokenize.py]: `ASYNC` and `AWAIT` should only be imported on Python v3.5 and above.

**Comment by gforsyth** [comment]
Ready for another look, @scopatz -- and I guess AppVeyor will keep failing since the windows tests are... well... failing

**Comment by scopatz** [comment]
Thanks @gforsyth!

**Pull request merged**